contracts/Executor.sol

```diff
@@ -86,7 +86,23 @@ contract Executor {
             }
 
             require(success, "Call failed");
-            state = state.writeOutputs(bytes1(command << 88), outdata);
+
+            if (
+                flags & COMMAND_TUPLE_RETURN != 0 &&
+                bytes1(command << 88) != 0xff
```

Comment:
We should probably use a constant like `NO_VALUE` instead of `0xff` here and elsewhere.